src/backend/lima.ts

```diff
@@ -177,11 +178,13 @@ interface SudoCommand {
 }
 
 const console = Logging.lima;
+const dependenciesYAMLPath = path.join(paths.resources, 'dependencies.yaml');
```

Comment:
Please just import it (like we do `DEFAULT_CONFIG`) instead; that compiles it into a plain JS object, so we don't need to include it in `electron-builder.yml`.

We don't have `js-yaml-loader` in the build scripts, though (we just run things through `ts-node`), so we can't do that in `utils/dependencies.ts`.